app/lib/features/comments/widgets/add_comment_widget.dart

```diff
@@ -111,6 +114,10 @@ class _AddCommentWidgetState extends ConsumerState<AddCommentWidget> {
     if (success) {
       _commentController.clear();
       showSendButton.value = false;
+      final postCommentFn = widget.postCreateComment;
+      if (postCommentFn != null) {
+        postCommentFn();
```

Comment:
What's the purpose of this function? For executing create operation? Why it's intended this way?

Reply:
because if you do `if (widget.postCreateComment) { widget.postCreateComment(); }` the linter is falsy complaining that this requires a `!` and I wanted to avoid that bang...